packages/new-editor/src/components/editor/variables/detail/EnumValues.tsx

```diff
@@ -24,7 +24,11 @@ export const PossibleEnumValues = ({ selectedValue: value, values, onChange }: P
   ];
   const meta = {
     updateData: (rowId: string, _columnId: string, value: unknown) => {
```

Comment:
I've changed the updateData interface in on of the latest ui-components updates. 
So the value should now be typed as string. Then you don't have to cast it manually.